x-pack/plugins/security_solution/public/entity_analytics/components/risk_score_useful_links_section.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { EuiLink, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { LinkAnchor } from '@kbn/security-solution-navigation/links';
+import { SecurityPageName } from '@kbn/security-solution-navigation';
+import styled from '@emotion/styled';
+import { euiThemeVars } from '@kbn/ui-theme';
+import * as i18n from '../translations';
+import { RiskInformationFlyout } from './risk_information';
+
+const StyledList = styled.ul`
```

Comment:
We should be moving away from styled components according to EUI team e-mails:

> EUI update (95.12 - 97.0, Emotion next steps, and color mode settings)
...
❓What about styled-components?
We also highly recommend Kibana engineers start moving away from styled-components. In addition to a Sass deprecation, a plan to deprecate and remove the use of styled-component in Kibana is in the works as well.


Read more https://github.com/elastic/eui/discussions/6828#discussioncomment-6076157